frb_example/pure_dart/rust/src/api.rs

```diff
@@ -725,3 +725,36 @@ impl ConcatenateWith {
         Ok(())
     }
 }
+
+#[derive(Debug, Clone)]
+#[frb(freezed)]
+pub enum Speed {
+    Unknown,
+    GPS(f64),
+}
+
+#[derive(Debug, Clone)]
+#[frb(freezed)]
+pub enum KilometerPoint {
+    Unknown,
+    SNCF(f64),
+}
+
+#[derive(Debug, Clone)]
+#[frb(freezed)]
+pub enum API {
+    Speed(Box<Speed>),
+    KilometerPoint(Box<KilometerPoint>),
+}
+
+pub fn handle_api(api: API) -> anyhow::Result<()> {
+    match api {
+        API::Speed(v) => {
+            println!("speed changed: {:#?}", *v);
+        }
+        API::KilometerPoint(v) => {
+            println!("kilometer point changed: {:#?}", *v);
+        }
+    };
+    Ok(())
+}
```

Comment:
I am not sure, shall we use names that clearly describe what we are testing, instead of these. Especially the "API"...

Reply:
Renamed in 04c39675717bb02c7183e3f4ee05738fde578fb8 : let me know if it's ok.

Reply:
Looks better :)